src/pages/ProductDetails/components/ProductInfo.jsx

```diff
@@ -32,16 +33,16 @@ const ProductInfo = ({ courtData }) => {
         <BoxFlex>
           <Point>
             <img src="/images/ProductDetail/실외 코트.png" />
-            <span>{{ type } > 3 ? "실내" : "실외"} 코트</span>
+            <span>{{ isIndoor } === 1 ? "실내" : "실외"} 코트</span>
```

Comment:
```suggestion
            <span>{ isIndoor === 1 ? "실내" : "실외"} 코트</span>
```
이 부분은 꼭 중괄호를 씌우지 않아도 될 것 같습니다